source/includes/prestashop/configure.md

```diff
@@ -6,14 +6,30 @@ To share your data with ShopRunBack, you must take the [authentication token of
 
```

Comment:
you must use

Reply:
remove the FR part of the URL, the user gonna be rejected to its preferred language